frontend/src/views/WheresMyGene/common/constants.ts

```diff
@@ -1 +1,3 @@
 export const HEATMAP_CONTAINER_ID = "heatmap-container-id";
+export const FMG_GENE_COUNT_LIMIT = 25;
```

Comment:
Good thinking, but the gene count limit is determined as a parameter sent via the request body to the marker genes endpoint. So this is not necessary and duplicating logic.

TL;DR - you can get rid of this and don't need to `.slice(0,FMG_GENE_COUNT_LIMIT)`